sky/resources.py

```diff
@@ -363,11 +363,14 @@ def _try_validate_accelerators(self) -> None:
 
         # Validate whether accelerator is available in specified region/zone.
         acc, acc_count = list(acc_requested.items())[0]
+        # Fractional accelerators are temporarily bumped up to 1.
+        if 0 < acc_count < 1:
+            acc_count = 1
```

Comment:
Should we do floor(acc_account) to handle e.g., 1.2?

Reply:
To my understanding, the current master is rejects any non-integer resources with count > 1. And the current PR works the same. Do you think we need to support such cases?

Reply:
I see; we can defer.